pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -491,6 +491,35 @@ func Convert_v1_FeatureHyperv_To_api_FeatureHyperv(source *v1.FeatureHyperv, hyp
 	return nil
 }
 
+// This function parses the SRIOV-VF-PCI-ADDR variable that is set by SR-IOV
+// device plugin listing PCI IDs for devices allocated to the pod. The format
+// is as follows:
+//
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
+func getSRIOVPciAddresses() []string {
```

Comment:
How do we know to correlated the interface to the allocated device? Is the order guaranteed here?

Reply:
We don't with current SR-IOV DP. With multiendpoints branch merged, each device pool will get its own variable: https://github.com/intel/sriov-network-device-plugin/pull/26

Reply:
I see. Feels like everything is up in the air. 
However, I do understand that this is a moving target...